README.md

```diff
@@ -1,3 +1,20 @@
+![Torrust Organization Header Image](img/2022_06_27-Torrust_Org_Title.png)
+
 # Torrust Enhancement Proposals
 
+Welcome!
 This repository contains Torrust's enhancement proposals.
+
+## What are TEPS and their purpose?
+
+TEPS, stands for Torrust Enhancement Proposals. If you have an idea for a new feature, a suggestion for improvement, think that the project could benefit from something, go ahead and tell us.
+
+We have created a TEP-Template for anyone to fill out and submit as
+
+Python Enhancement Proposals
```

Comment:
I think you forgot to edit/remove everything below this line that links to PEP and Python instead of TEP and Torrust.

Reply:
I did not save it....